internal/controller/git/base_repo.go

```diff
@@ -232,13 +232,20 @@ func (b *baseRepo) buildCommand(command string, arg ...string) *exec.Cmd {
 func (b *baseRepo) buildGitCommand(arg ...string) *exec.Cmd {
 	cmd := b.buildCommand("git", arg...)
 	cmd.Env = append(cmd.Env, fmt.Sprintf("GIT_SSH_COMMAND=ssh -F %s/.ssh/config", b.homeDir))
+
 	if b.creds != nil && b.creds.Password != "" {
-		cmd.Env = append(
-			cmd.Env,
-			"GIT_ASKPASS=/usr/local/bin/credential-helper",
-			fmt.Sprintf("GIT_PASSWORD=%s", b.creds.Password),
-		)
+		cmd.Env = append(cmd.Env, "GIT_ASKPASS=/usr/local/bin/credential-helper")
+		cmd.Env = append(cmd.Env, fmt.Sprintf("GIT_PASSWORD=%s", b.creds.Password))
```

Comment:
This was more efficient as previously written, as it resized the slice just once. Now it's being resized twice. It's a quite a small difference, but if there isn't a good reason for the change, I'd rather see it left alone.